packages/input-number/__tests__/input-number.spec.ts

```diff
@@ -1,32 +1,173 @@
-import { mount } from '@vue/test-utils'
 import InputNumber from '../src/index.vue'
-import { ref } from  'vue'
+import { mount } from '@vue/test-utils'
+import { ref } from 'vue'
 
 const _mount = options => mount({
   components: {
     'el-input-number': InputNumber,
   },
   ...options,
 })
-describe('InputNumber.vue', () => {
-  test('create', async () => {
+
+describe('input-number.vue', () => {
+  test('create', () => {
+    const wrapper = _mount({
+      template: '<el-input-number />',
+    })
+    expect(wrapper.find('input').element.value).toEqual('0')
+    expect(wrapper.find('.el-input-number').exists()).toBe(true)
+  })
+  test('modelValue', () => {
     const wrapper = _mount({
-      template: `
-        <el-input-number
-          label="描述文字"
-          v-model="num">
-        </el-input-number>
-      `,
+      template: '<el-input-number :model-value="inputText" />',
       setup() {
-        const num = ref(1)
+        const inputText = ref(1)
 
+        return {
+          inputText,
+        }
+      },
+    })
+    expect(wrapper.find('input').element.value).toEqual('1')
+  })
+  test('min', () => {
+    const wrapper = _mount({
+      template: '<el-input-number :min="3" model-value="1" />',
+    })
+    expect(wrapper.find('input').element.value).toEqual('3')
+  })
+  test('max', () => {
+    const wrapper = _mount({
+      template: '<el-input-number :max="3" model-value="5" />',
+    })
+    expect(wrapper.find('input').element.value).toEqual('3')
+  })
+  test('step increase and decrease', async () => {
+    const wrapper = _mount({
+      template: '<el-input-number :step="3" v-model="num" />',
+      setup() {
+        const num = ref(2)
         return {
           num,
         }
       },
     })
-
-    const InputNumberElm = wrapper.find('input')
-    expect(InputNumberElm.exists()).toBe(true)
+    await wrapper.find('.el-input-number__increase').trigger('keydown.enter')
+    expect(wrapper.find('input').element.value).toEqual('5')
+    await wrapper.find('.el-input-number__decrease').trigger('keydown.enter')
+    expect(wrapper.find('input').element.value).toEqual('2')
+  })
+  test('step-strictly', async () => {
+    const wrapper = _mount({
+      template: '<el-input-number :step-strictly="true" :step="2" v-model="num" />',
+      setup() {
+        const num = ref(0)
+        return {
+          num,
+        }
+      },
+    })
+    await wrapper.find('input').setValue(3)
+    expect(wrapper.find('input').element.value).toEqual('4')
+  })
+  test('precision', async () => {
+    const wrapper = _mount({
+      template: '<el-input-number :precision="2" v-model="num" />',
+      setup() {
+        const num = ref(0)
+        return {
+          num,
+        }
+      },
+    })
+    await wrapper.find('input').setValue(1.1111111111)
```

Comment:
Maybe we need to test whether manually setting value equals to `1.11` or something? This should be tested, correct?

Reply:
got it, already correct it